pypesto/objective/priors.py

```diff
@@ -0,0 +1,264 @@
+import scipy.stats
+import numpy as np
+from typing import Optional, Union, Iterable, Dict, Callable, List, Tuple
+from copy import deepcopy
+import math
+
+from .function import Objective
+from .aggregated import AggregatedObjective
+
+
+class Priors(AggregatedObjective):
+    """
+    Handles prior distributions.
+
+    Consists basically of a list of individual priors,
+    given in self.objectives.
+    """
+
+    def __init__(self,
+                 prior_list: List[Objective],
+                 x_names: List[str] = None):
+
+        super().__init__(prior_list, x_names)
+
+
+class ParameterPriors(Objective):
+    """
+    Single Parameter Prior.
+
+
+    prior_list has to contain dicts of the format format
+    {'index': [int], 'density_fun': [Callable], 'density_dx': [Callable], 'density_ddx': [Callable]}
+
+
+    Note:
+    -----
+    All callables should correspond to (log)densities and are internally
+    multiplied by -1, since pyPESTO performs minimization...
+    """
+
+    def __init__(self,
+                 prior_list: list):
```

Comment:
document args in docstring
```suggestion
                 prior_list: List[Dict]):
```